app/src/main/java/org/mifos/selfserviceapp/ui/activities/HomeActivity.java

```diff
@@ -119,4 +125,27 @@ protected void onDestroy() {
         super.onDestroy();
         preferencesHelper.clear();
     }
+
+    @Override
+    public void onBackPressed() {
+        if (doubleBackToExitPressedOnce && stackCount() == 0) {
+            HomeActivity.this.finish();
+            return;
+        }
+        this.doubleBackToExitPressedOnce = true;
+        Toaster.show(view , "double tap on Back Button to exit");
```

Comment:
Please remove the hard coded string to strings.xml. and pass this `findViewById(android.R.id.content)` instead of view